menuinst/windows/win32.py

```diff
@@ -113,6 +113,9 @@ def quoted(s):
     """
     # strip any existing quotes
     s = s.strip(u'"')
+    # don't add quotes for minus or leading space
+    if u'-' in s[:1] or u' ' in s[:1]:
```

Comment:
more compact, pythonic way to write this:

```suggestion
    if s[0] in (u'-', u' '):
```